Makefile

```diff
@@ -310,9 +309,9 @@ go_mockgen: ## Use `mockgen` to generate mocks used for testing purposes of all
 	go generate ./x/proof/types/
 	go generate ./x/tokenomics/types/
 	go generate ./pkg/client/interface.go
```

Comment:
As an improvement, why not do something like: `find . -name "interface.go | xargs -I {} go generate {}"`

(I didn't test this so bash is from memory but I'm sure ChatGPT can help)

Reply:
@red-0ne you 👍ed the comment, re-requested a review, but the code is still the same?